community_supplied/Protocols/Bgp/check-bgp-summary-netconf.rule

```diff
@@ -0,0 +1,174 @@
+            */
+            function neighbor-check {
+                path BGPneighbor.py;
+                method neighbor;
+                argument neighbor_list {
+                    mandatory;
+                }
+                argument peer_state {
+                    mandatory;
+                }
+            }
+            /*
+            * Sensor configuration to get data from network devices.
+            */
+            sensor bgp-sensor {
+                synopsis "BGP configuration iAgent sensor definition";
+                description "Netconf rpc get-bgp-summary-information to collect data from network device";
+                iAgent {
+                    file BgpSummary.yml;
+                    table BgpSummaryTable;
+                    frequency 60s;
+                }
+            }
+            /*
+            * Fields defined using formulae by user-defined function
+            */
+            field neighbor-check {
+                formula {
+                    user-defined-function {
+                        function-name neighbor-check;
+                        argument neighbor_list "@neighbor";
+                        argument peer_state "$peer-address";
+                    }
+                }
+                type integer;
+            }
+            /*
+            * Field defined using sensor path. Map the longer sensor names
+            * to the shorter field names used in the rules.
+            */
+            field peer-address {
+                sensor bgp-sensor {
+                    where "peer-address =~ /^{{peer-address-var}}$/";
+                    path peer-address;
+                }
+                type string;
+                description "Neighbor address";
+            }
+            field peer-state {
+                sensor bgp-sensor {
+                    path peer-state;
+                }
+                type string;
+                description "Neighbor state";
+            }
+            /*
+            * Anomaly detection logic.
+            */
+            trigger bgp-neighbor {
+                synopsis "BGP neighbors";
+                description "Sets health based on the change in BGP neighbor and neighbor state";
+                frequency 90s;
+                /*
+                * Sets color to green and sends out an anomaly notification when
+                * neighbor check ($neighbor-check) is equal to 1 and neighbor
+                * state is equal to Established.
+                */
+                term neighbor-is-configured {
+                    when {
+                        equal-to "$neighbor-check" 1;
+                        matches-with "$peer-state" Established;
+                    }
+                    then {
+                        status {
+                            color green;
+                            message "Neighbor address ($peer-address) is configured. Neighbor state is Established.";
```

Comment:
change from "Neighbor address ($peer-address) is configured. Neighbor state is Established." to
"Neighbor address ($peer-address) is configured. Neighbor state is $peer-state."